alternator/server.cc

```diff
@@ -203,7 +203,7 @@ class local_nodelist_handler : public gated_handler {
         // just the list of live nodes in this DC needs more elaborate code:
         auto& topology = _proxy.get_token_metadata_ptr()->get_topology();
```

Comment:
Unrelated: need to hold this across usage.

Reply:
the usage of topology here is currently synchronous so it's not strictly necessary.